litmus-portal/backend/graphql-server/graph/project.graphqls

```diff
@@ -0,0 +1,12 @@
+type Project{
```

Comment:
We are having all the type declarations in the schema.graphqls, and if we are separating these the related queries/mutations should also be separated.

Reply:
Yes but there can be only one type struct for `Query`, `Mutation` and `SUbscription` . So we need to have all of them under one roof . Instead we can have the `type` and their resolvers in seperate files. We can refactor tye code in the coming releases